.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,7 +26,8 @@
     - Sample source(s):
       - [URL to each sample source, if applicable]
     - Sample license(s):
-  - [ ] I have included a syntax highlighting grammar.
+  <!-- Update the Lightshow URLs below to show the grammar in action if you included one. -->
+  - [ ] I have included a syntax highlighting grammar: https://github-lightshow.herokuapp.com/
   - [ ] I have included a change to the heuristics to distinguish my language from others using the same extension.
```

Comment:
Not related to this PR, but this caught my eye while inspecting the diff:

> I have ~~**included a change to**~~ the heuristics …

The words `included a change to` can be more concisely expressed by

> I have ***updated*** the heuristics …

Which flows much nicer:

> I have updated the heuristics to distinguish my language from others using the same extension.